### PR TITLE
Update Node.js docs to include new env vars

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

Update the Node.js docs to mention the new `BP_NODE_VERSION` and `BP_NODE_OPTIMIZE_MEMORY` environment variables, and mentions the impending deprecation of setting values through the `buildpack.yml`.

This PR should not be merged until https://github.com/paketo-buildpacks/node-engine/pull/240 is merged, so that the new variables are in the buildpack. This is a part of https://github.com/paketo-buildpacks/node-engine/issues/209

## Use Cases

Keeps docs up to date

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.

### CODE REVIEW

File: content/docs/buildpacks/language-family-buildpacks/nodejs.md [hunk 1]
```diff
@@ -204,6 +208,20 @@ will be chosen based on a priority list which reflects the order of the list
 above with `server.js` being the highest priority and  `index.js` being the
 lowest.
 
+### Using BP_LAUNCHPOINT
+THIS IS NODE START SPECIFIC
```
Comment (kvedurmu) [inline content/docs/buildpacks/language-family-buildpacks/nodejs.md:212]: Could you rephrase this/remove? I think the explanation below is sufficient.

**Review by kvedurmu — CHANGES_REQUESTED** [review]

File: content/docs/buildpacks/language-family-buildpacks/nodejs.md (hunk 1 above)
Comment (ForestEckhardt) [reply content/docs/buildpacks/language-family-buildpacks/nodejs.md:212]: Yeah that shouldn't be there

**Review by kvedurmu — APPROVED** [review]
